#goal
---
#result
geometry_msgs/Point list_of_odoms
---
#feedback
float32 current_total